backend/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "npm run prod",
-    "server": "babel-node ./src/index.js",
+    "server": "babel-node -r ../node_modules/dotenv/config ./src/index.js dotenv_config_path=./config/dev.env",
```

Comment:
why?

Reply:
`nodemon` and `babel-node` were not setting up the environment variables before starting the server. It was not preloading `dotenv` config files. So I had to explicitly set -r flag to preload the variables

Reply:
Resolved now. Now I am loading dynamically using dotenc